README.md

```diff
@@ -172,6 +173,15 @@ Calling `bridge_rewards` on L2 token bridge results in:
   self by calling `claimRewards` on the `IncentivesController` contract.
 - The rewards are then transferred to the L1 recipient.
 
+### ATokens deposit cancellation
+If the L2 message consumption is unsuccessful (this might happen due to a bug in the dApps’s Cairo contract). The user would lose custody over his `aTokens` forever! 
```

Comment:
This sentence implies that our code may contain bugs. This is unnecessary bad publicity. Also, no need to add "!" as the end, we're should describe the code without sharing emotion. We could write instead:

`If L1 -> L2 message consumption is unsuccessful, the user would lose custody over his `aTokens` forever. It can happen if the sequencer is down, or if the message is misconfigured.`